app/services/usps_in_person_proofing/enrollment_validator.rb

```diff
@@ -0,0 +1,88 @@
+module UspsInPersonProofing
+  class EnrollmentValidator
+    SUPPORTED_FIELDS = [
+      :first_name,
+      :last_name,
+      :address1,
+      :address2,
+      :city,
+    ].freeze
+
+    def initialize(transliterator = Transliterator.new)
+      @transliterator = transliterator
+    end
+
+    # Validate that transliteratable fields from the state ID form
+    # can be transliterated to contain acceptable characters.
+    #
+    # @param [Hash] fields Name/value pairs of fields to validate
+    # @option fields [String,#to_s] :first_name
+    # @option fields [String,#to_s] :last_name
+    # @option fields [String,#to_s] :address1
+    # @option fields [String,#to_s] :address2
+    # @option fields [String,#to_s] :city
+    # @return [Hash,nil] Field names mapped to error messages, or nil if there were no issues
+    def validate(fields)
```

Comment:
Why not use kwargs here? and build a hash from that?
```suggestion
    def validate(first_name: nil, last_name: nil, address1: nil, address2: nil, city: nil)
      fields = { first_name:, last_name:, address1:, address2: city: }.select(&:present?)
```

Reply:
I'll take a look at the impact implementing this change would have on the controller. I was trying here to keep logic about which fields get validated relatively isolated to this class, but your suggested usage could be easier to work with.